src/main/java/uk/gov/ons/census/action/model/entity/FulfilmentsToSend.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ons.census.action.model.entity;
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.Data;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.hibernate.annotations.TypeDefs;
+
+@Entity
+@TypeDefs({@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)})
+@Data
+public class FulfilmentsToSend {
```

Comment:
I think it should be `FulfilmentToSend` (singular) because it's just one fulfilment